src/ngraph/runtime/generic_cpu/gcpu_executable.hpp

```diff
@@ -437,6 +439,30 @@ class ngraph::runtime::gcpu::GCPUExecutable : public Executable
                                        broadcast_axes);
             break;
         }
+        case OP_TYPEID::BroadcastDistributed: {
+#ifdef NGRAPH_DISTRIBUTED_ENABLE
+            Distributed dist;
+            int Rank_ID;
+            Rank_ID = dist.get_rank();
+            if (Rank_ID == 0)
```

Comment:
Just a question - why do we need 2 bcast ops (broadcastrecv and broadcastsend) which differ only in arguments name?

Reply:
Thanks @AlekseyMarchuk for catching this! Yes, you are right, we only need one function for this.